employees2.sql

```diff
@@ -0,0 +1,116 @@
+CREATE TABLE pracownik (
+	id bigint primary key auto_increment NOT NULL,
+    imie varchar(50) NOT NULL,
+    nazwisko varchar(50) NOT NULL,
+    stanowisko varchar(200) NOT NULL,
+    data_urodzenia date NOT NULL,
+    wyplata double NOT NULL
+);
+
+INSERT INTO pracownik (imie, nazwisko, stanowisko, data_urodzenia, wyplata) 
+VALUES ('Anna', 'Abacka', 'Sekretarka', '1992-05-11', 3000.50),
+('Jan', 'Zyga', 'Developer', '1980-09-12', 8000.50),
+('Maria', 'Bratek', 'UI', '1988-09-16', 6000.50),
+('Genowefa', 'Pigwa', 'Developer', '1996-09-14', 5000.00),
+('Marcin', 'Mala', 'Developer', '1985-09-14', 9000.00),
+('Nadia', 'Kot', 'Menager','1980-09-14', 15000.00);
+
+UPDATE pracownik SET data_urodzenia='1996-09-14' WHERE id=13;
+
+SELECT * FROM pracownik ORDER BY nazwisko;
+
+SELECT * FROM pracownik WHERE stanowisko='Developer';
+
+SELECT * FROM pracownik WHERE EXTRACT(YEAR FROM data_urodzenia) < 1990;
```

Comment:
Tutaj też nie poprawiony algorytm - sprawdź jeszcze raz to co napisałem Ci na slacku o tym